m4_sessions/1708-inning/group_four.md

```diff
@@ -12,19 +12,53 @@ Privacy: What it means for devs and their users
 ### Session Outline:
 *This session is about privacy in the scope of Corporate snooping, not Government Surveillance. We're limiting our scope to allow more targeted discussions.
 
-- Intros
+### Pre-class/Prep
+[ ] Jimmy signs into my creepy app
+[x] Find one more video about creepy privacy stuff
+[x] Find a couple definitions of privacy
+[x] Find article about innocuous data
 
-- Activities
-* short video about creepy stuff?
-* maybe use my creepy app?
+# (10 min) Intros
+- Name, mod, FE/BE
+- What super power would you have?
 
-- Discussions
-* What is privacy? What does it mean to you?
-* What are you willing to give away? Where do you draw the line for what you're willing to give away?
-* What is innocuous data? How does giving away small bits of data add up?
-* How privacy can impact your app use
-* Do users know what they're signing up for?
-* Emergent Tech and Privacy
+# Activities
+* (5 min) short videos about creepy stuff
+```
+https://www.youtube.com/watch?v=7bXJ_obaiYQ
+https://www.youtube.com/watch?v=bqWuioPHhz0
```

Comment:
I like the second video more that the first.  The first isn't as powerful without context of what is happening in that movie.